starlight_help/src/content/docs/moving-from-discord.mdx

```diff
@@ -0,0 +1,125 @@
+
+The following steps are described in more detail below:
+
+<Steps>
+  1. [Create your organization](#create-your-organization).
+  1. [Sign up for a plan](#sign-up-for-a-plan).
+  1. [Configure your organization](#configure-your-organization).
+  1. [Review and update communication
+     policies](#review-and-update-communication-policies).
+  1. [Prepare users for the transition](#prepare-users-for-the-transition).
+  1. [Invite users to join](#invite-users-to-join).
+</Steps>
+
+Each organization is unique, but we hope these common practices will help you
+think through the transition process in your own context.
+
+## Create your organization
+
+<Steps>
+  1. If you plan to self-host, [set up your Zulip
+     server](https://zulip.readthedocs.io/en/stable/production/install.html). You
+     can self-host Zulip directly on Ubuntu or Debian Linux, in
+     [Docker](https://github.com/zulip/docker-zulip), or with prebuilt images for
+     [Digital Ocean](https://marketplace.digitalocean.com/apps/zulip) and
+     [Render](https://render.com/docs/deploy-zulip).
+  1. Create a Zulip organization [on Zulip Cloud](https://zulip.com/new/) or
+     [on your self-hosted
+     server](https://zulip.readthedocs.io/en/stable/production/install.html#step-3-create-a-zulip-organization-and-log-in).
+     You can create a new Zulip Cloud organization in less than two minutes.
+     Setting up a self-hosted server will take a bit longer, but is easy to
+     do with Zulip's [robust
+     installer](https://zulip.readthedocs.io/en/stable/production/install.html).
+</Steps>
+
+## Sign up for a plan
+
+<SignUpForAPlan />
+
+## Configure your organization
+
+<FlattenedSteps>
+  <ConfigureYourOrganization />
+
+  1. [Create channels](/help/create-channels). Zulip's
+     [topics](/help/introduction-to-topics) give each conversation its own
+     space, so one channel per team should be enough to get you started.
+  1. To recreate your Discord bots in Zulip, check out Zulip's [native
+     integrations](/integrations), and the [guide](/api/writing-bots) on writing
+     your own interactive bots.
+</FlattenedSteps>
+
+## Review and update communication policies
+
+<CommunicationPolicies />
+
+* If you've been maintaining a forum in addition to your Discord server, it's
+  common to discontinue it when moving to Zulip. Conversations in Zulip are
+  organized enough to fulfill the role of a forum, and can be
+  [configured](/help/public-access-option) for public access if desired.
+* Zulip makes it easy to find conversations and follow up. To avoid disrupting
+  focus, @-mentions in Zulip should generally be reserved for time-sensitive
+  messages. [Silent
+  mentions](/help/mention-a-user-or-group#silently-mention-a-user) make it easy
+  to refer to someone without calling for their attention.
+* Because conversations are specific to a topic, there's generally no need to
+  reply to a specific message -- it will be clear enough what you're responding
```

Comment:
"reply" should probably be a link to our quote-reply feature for now. I would also be a bit less strong in the wording. Something like:

> Many situations where one would have replied to specific message in Discord are better handled in Zulip by just sending a regular message to the the topic.